### DISCUSSION TITLE
Retrieving previous successful form submission data as flash

### DISCUSSION THREAD

**Discussion by hsh3nl** [solspace/craft-express-forms#55] [Feature Requests]
### Description
On successful form submission, is there a way to retrieve the latest successful form submission data (field values) as part of the flash?

From the documentation, it seems that the only data that can be retrieved after a successful form submission is whether the form was successful submitted (boolean) https://craft.express/forms/v1/templating/#flash-success-message).

### Additional info

- Express Forms edition & version: Express Forms Lite 1.0.6
- Craft version: 3.5.5
- PHP version: 7.2.32
- Database driver & version: MySQL 5.5.5
- Other plugins & versions (if applicable):

**Comment by kjmartens** [comment]
@hsh3nl Sorry for the delay... currently there's no way to do this. While this isn't possible in Freeform either, you can access submission data in front end templates and have the form success return to a template that includes the submission data. If you'd like to consider that, you can check out the guide for Freeform here:
https://docs.solspace.com/craft/freeform/v3/guides/submission-data-success.html
